core/src/org/pentaho/di/core/row/value/ValueMetaBase.java

```diff
@@ -2708,7 +2708,7 @@ public void writeMeta( DataOutputStream outputStream ) throws KettleFileExceptio
       writeString( outputStream, dateFormatLocale != null ? dateFormatLocale.toString() : null );
 
       // date time zone?
-      writeString( outputStream, dateFormatTimeZone != null ? dateFormatTimeZone.toString() : null );
+      writeString( outputStream, dateFormatTimeZone != null ? dateFormatTimeZone.getID() : null );
```

Comment:
getID() returns a normalized timezone string, which is good. However will it break compatibility with existing transformations? If so we need to get Doc and PM involved